armi/physics/fuelCycle/fuelHandlers.py

```diff
@@ -1397,10 +1397,16 @@ def dischargeSwap(self, incoming, outgoing):
         # which, coincidentally is the same time we're at right now at BOC.
         self.r.core.removeAssembly(outgoing)
 
+        # adjust the assembly multiplicity so that it doesnt forget how many it really
+        # represents
+        outgoing.p.multiplicity = len(loc.getSymmetricEquivalents()) + 1
```

Comment:
that's kind of a lot of work just to get the number. Maybe add explanation for why this was the best choice (related to center assem vs. edge assem vs. normal assem?